build.js

```diff
@@ -0,0 +1,23 @@
+const esbuild = require("esbuild");
+const { targets } = require("./esbuild.config.js");
+
+async function build() {
+  for (const target of targets) {
+    try {
+      await esbuild.build(target);
```

Comment:
Is it too memory intensive if we build all of these in parallel?

Reply:
Seems to be working okay both locally and in CI.